code/modules/client/preferences.dm

```diff
@@ -177,7 +177,16 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 			dat += "<a href='?_src_=prefs;preference=name;task=input'>[active_character.real_name]</a><BR>"
 
 			if(!(AGENDER in active_character.pref_species.species_traits))
-				dat += "<b>Gender:</b> <a href='?_src_=prefs;preference=gender'>[active_character.gender == MALE ? "Male" : "Female"]</a><BR>"
+				var/dispGender
+				if(active_character.gender == MALE)
+					dispGender = "Male"
+				else if(active_character.gender == FEMALE)
+					dispGender = "Female"
+				else
+					dispGender = "Other"
+				dat += "<b>Gender:</b> <a href='?_src_=prefs;preference=gender'>[dispGender]</a><BR>"
+				if(active_character.gender == PLURAL || active_character.gender == NEUTER)
+					dat += "<b>Body Model:</b><a href='?_src_=prefs;preference=body_model'>[active_character.features["body_model"] == MALE ? "Masculine" : "Feminine"]</a><BR>"
```

Comment:
Personally I don't think hiding the body model pref behind the other gender makes sense. Since the game will use they for the Other gender always, you can't have a character that uses he/him but has the feminine body model or the reverse.

It should certainly default when you change your gender to the matching type but it shouldn't entirely lock you of the option - OR the actual medrecords gender and pronouns should be separated. For now though the previous solution is easier.

Reply:
> It should certainly default when you change your gender to the matching type but it shouldn't entirely lock you of the option - OR the actual medrecords gender and pronouns should be separated. For now though the previous solution is easier.

Soooo...
Keep it this way or revert it to how it was previously?

Reply:
There should be a pronoun option in the other category if its not out of scope, but I'm fine with it this way.

Reply:
The problem is the game makes no distinction between gender and pronouns, as gender functionally is the pronouns. Separating them would be a huge PITA

Reply:
It would probably need an additional pref which could be a bit of a pain, yea. I'm fine to accept that as something that can just be done in a future PR.